base/scripts/run_app.sh

```diff
@@ -4,14 +4,14 @@ if [ -d /bundle ]; then
   cd /bundle
   tar xzf *.tar.gz
   cd /bundle/bundle/programs/server/
-  npm i
+  npm install --unsafe-perm
```

Comment:
Without `--unsafe-perm` it did not work, this solution was provided in in issue from the Meteor repository